File: src/main/java/gherkin/GherkinDialect.java
package gherkin;

public class GherkinDialect {
    public String[] getFeatureKeywords() {
        return new String[]{"Feature"};
    }

    public String[] getScenarioKeywords() {
        return new String[]{"Scenario"};
    }

    public String[] getStepKeywords() {
        return new String[]{"Given ", "When ", "Then "};
    }
}

File: javascript/lib/gherkin/gherkin_dialect.js
module.exports = function GherkinDialect() {
  this.featureKeywords = ['Feature:'];
  this.backgroundKeywords = ['Background:'];
  this.scenarioKeywords = ['Scenario:'];
  this.scenarioOutlineKeywords = ['Scenario Outline:', 'Scenario Template:'];
  this.examplesKeywords = ['Examples:', 'Scenarios:'];
  this.stepKeywords = ['Given ', 'When ', 'Then ', 'And ', 'But ', '* '];
};
